app/routes/profile/index.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model(params) {
+    return Ember.RSVP.hash({
```

Comment:
I haven't finished everything here yet 😄 Still need to flesh out the index and favorites routes when I get the time